pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+trio_mode=True
+log_cli=1
```

Comment:
let's comment this out since ideally we don't have the live logging to console by default 😉